client/coral-admin/src/reducers/comments.js

```diff
@@ -23,13 +23,13 @@ const initialState = Map({
 // Handle the comment actions
 export default (state = initialState, action) => {
   switch (action.type) {
-  case 'COMMENTS_MODERATION_QUEUE_FETCH': return state.set('loading', true);
-  case 'COMMENTS_MODERATION_QUEUE_FETCH_SUCCESS': return replaceComments(action, state);
-  case 'COMMENTS_MODERATION_QUEUE_FAILED': return state.set('loading', false);
-  case 'COMMENT_STATUS_UPDATE': return updateStatus(state, action);
-  case 'COMMENT_FLAG': return flag(state, action);
-  case 'COMMENT_CREATE_SUCCESS': return addComment(state, action);
-  case 'COMMENT_STREAM_FETCH_SUCCESS': return replaceComments(action, state);
+  case actions.COMMENTS_MODERATION_QUEUE_FETCH: return state.set('loading', true);
+  case actions.COMMENTS_MODERATION_QUEUE_FETCH_SUCCESS: return replaceComments(action, state);
```

Comment:
yes, definitely. I'll probably save it for another PR. I'm just trying to remove the talk-adapter weirdness in this one.